### PR TITLE
Braeden McNulty HW1

### PR DESCRIPTION
**Pull request by braedenmc1**
Most of the commands (e.g. bolding, italics, adding links) were fairly straight forward. The hardest part was creating lists due to the particular nature of the formatting. I talked to Colton after trying a few different syntaxes, and he suggested I added some spaces between the code lines. That worked well, and I am confident I can replicate it in the future. I had some difficulty translating my image into Markdown. Using advice from Deb and Colton, I was able to get it to work.

### CODE REVIEW

**Review by onjas-6 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |        x      |            |
| **Presentation**            |    x       |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |   x        |              |            |

Remarks:
* I think in many places you intend to show a heading, but the intention is now shown on the redered file online. '#' this doesn't work unless you place a space after it like this "# Heading."
* It's great that you incorporate the picture in the folder! It's a great method to show the picture without writing long URLs.
* I think it's great work because all the elements mentioned in the rubric are completed. Although there are some problems with heading, but I think that's fine!

**Review by leahvf — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |           |     X         |            |
| **Achievement, creativity** |      X     |              |            |
| **Ease of access**          |      X     |              |            |

Remarks:

I really enjoyed your page! I learned how to code the captioning that you put under the Star Wars photo. I'm not sure if you meant to leave the # in front the section about MAPPS, or if you meant it to be a header - if it's the latter its an easy fix, if it's the former you may want to consider keeping the hashtagged word lowercase.  Reaching out for help is an important part of the process, I'm glad you decided to share your experience.

**Comment by jongriffiths** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |       x    |              |            |

Remarks: 

Saying Revenge of the Sith is your favorite Star Wars movie is a bold claim, but I respect it.  I think the spaceship fight outside of Count Dooku's ship in RotS is a very underrated battle scene. I'm sure you've received a lot of pushback to your opinion. As Last Jedi defender, I'm something of a kindred spirit.  In terms of substantive feedback, just make sure you have a space between the # and the title you're trying to modify which was pointed out by the other commenters. Other than that, great job!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+
+1. Student Government
+2. First Year Orientation
+3. Admissions Tour Guide
+
+During my tenure on Centre's student government, I spearhaeded a project that led to a composting program with our dining services that is still in use today.
+
+#The MAPSS Program
+Within the MAPSS program, my thesis is about how religiously framing climate change threats and policy proposals affects the extent to which religious Americans percieve the issue and support policy outcomes. The project stemmed from my upbringing in Pikeville where Coal and Christianity are at the heart of political discussions. Broadly speaking, my interest in the politics of climate change and environmental protection rose from a conversation I had with a friend back home. He is an environmental engineer who dislkes the EPA. When talking to him about the health of local waterways, he is very concerned about corporate pollution. Yet, when environmental protection turns to climate change, he is more hesitant to support regulation.
```
Comment (bensoltoff) [inline README.md:12]: Add a blank line between the header `#The MAPSS Program` and your text. If you view the document on GitHub, the formatting does not appear as you intended

File: README.md [hunk 2]
```diff
@@ -1,2 +1,23 @@
-# hw01
-Repository for homework 01
+# **This is the Repository of Charles Braeden McNulty**
+My name is Braeden McNulty, a Political Science MAPSS student from [Pikeveille, KY.](http://pikevilleky.gov/)
+After graduating from Pikeville High School, I atteneded [Centre College](https://www.centre.edu/) in Danville, KY where I earned a BA in _Political Science_ and _History_. At Centre, I was involved numverous organizations that allowed me to share my love of my school with others. These include: 
+
+1. Student Government
+2. First Year Orientation
```
Comment (bensoltoff) [inline README.md:6]: You don't need to assign the numbers yourself. For example,

```
1. Student Government
1. First Year Orientation
1. Admissions Tour Guide
```

renders as

1. Student Government
1. First Year Orientation
1. Admissions Tour Guide

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      x        |            |
| **Coding strategy**         |           |    x          |            |
| **Presentation**            |       x    |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |        x   |              |            |

Remarks:

* Nice inclusion of the photo in your repository. If it was hosted on another website, you could just directly link to it

**Pull request closed without merging**